src/arch/aarch64/kernel/start.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Stefan Lankes, RWTH Aachen University
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use crate::arch::aarch64::kernel::serial::SerialPort;
+use crate::arch::aarch64::kernel::BootInfo;
+use crate::KERNEL_STACK_SIZE;
+
+static mut BOOT_STACK: [u8; KERNEL_STACK_SIZE] = [0; KERNEL_STACK_SIZE];
+
+/// Entrypoint - Initalize Stack pointer and Exception Table
+#[inline(never)]
+#[no_mangle]
+#[naked]
+pub unsafe extern "C" fn _start() -> ! {
+	asm!("ldr x1, {0}",
+		 "add x1, x1, {1}",
+		 "sub x1, x1, #0x10",	/*Previous version subtracted 0x10 from End, so I'm doing this too. Not sure why though */
```

Comment:
If you know why this subtraction is there, please update the comment I left here.

Reply:
I added a comment.